todo_app/app.py

```diff
@@ -8,4 +10,17 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+    
+    return render_template('index.html', items = items)
+
+
+@app.route('/item', methods=['POST']) 
+def add_item_to_list():
+
+    itemTitle = request.form.get('item-input')
+    if itemTitle != "":
+        add_item(itemTitle)
+        
+    
+    return redirect(url_for('index'))
```

Comment:
nice work finding the redirect and url_for!

Reply:
Thanks : )